templates/ui-package/tmp-package.json

```diff
@@ -42,7 +43,7 @@
     "@angular/router": "~9.1.4",
     "@labshare/ls-font": "^1.0.0",
     "@labshare/ngx-base-components": "^1.2.4",
-    "@labshare/ngx-core-services": "^2.1.4",
+    "@labshare/ngx-core-services": "2.1.3",
```

Comment:
Locking it to 2.1.3 doesn't seem ideal to me. Especially for a project template.

Reply:
Updated it to v3, and used .env file with empty string in `SERVICES_AUTH_URL` to fix it.